packages/contracts-bedrock/src/universal/StandardBridge.sol

```diff
@@ -391,63 +273,19 @@ abstract contract StandardBridge is Initializable {
-        return ERC165Checker.supportsInterface(_token, type(ILegacyMintableERC20).interfaceId)
-            || ERC165Checker.supportsInterface(_token, type(IOptimismMintableERC20).interfaceId);
-    }
-
-    /// @notice Checks if the "other token" is the correct pair token for the OptimismMintableERC20.
-    ///         Calls can be saved in the future by combining this logic with
-    ///         `_isOptimismMintableERC20`.
-    /// @param _mintableToken OptimismMintableERC20 to check against.
-    /// @param _otherToken    Pair token to check.
-    /// @return True if the other token is the correct pair token for the OptimismMintableERC20.
+    /**
+     * @notice Checks if the "other token" is the correct pair token for the OptimismMintableERC20.
+     *         Calls can be saved in the future by combining this logic with
+     *         `_isOptimismMintableERC20`.
+     *
+     * @param _mintableToken OptimismMintableERC20 to check against.
+     * @param _otherToken    Pair token to check.
+     *
+     * @return True if the other token is the correct pair token for the OptimismMintableERC20.
+     */
     function _isCorrectTokenPair(address _mintableToken, address _otherToken) internal view returns (bool) {
-        if (ERC165Checker.supportsInterface(_mintableToken, type(ILegacyMintableERC20).interfaceId)) {
-            return _otherToken == ILegacyMintableERC20(_mintableToken).l1Token();
-        } else {
-            return _otherToken == IOptimismMintableERC20(_mintableToken).remoteToken();
-        }
-    }
-
-    /// @notice Emits the ETHBridgeInitiated event and if necessary the appropriate legacy event
-    ///         when an ETH bridge is finalized on this chain.
-    /// @param _from      Address of the sender.
-    /// @param _to        Address of the receiver.
-    /// @param _amount    Amount of ETH sent.
-    /// @param _extraData Extra data sent with the transaction.
-    function _emitETHBridgeInitiated(
-        address _from,
-        address _to,
-        uint256 _amount,
-        bytes memory _extraData
-    )
-        internal
-        virtual
-    {
-        emit ETHBridgeInitiated(_from, _to, _amount, _extraData);
-    }
-
-    /// @notice Emits the ETHBridgeFinalized and if necessary the appropriate legacy event when an
-    ///         ETH bridge is finalized on this chain.
-    /// @param _from      Address of the sender.
-    /// @param _to        Address of the receiver.
-    /// @param _amount    Amount of ETH sent.
-    /// @param _extraData Extra data sent with the transaction.
-    function _emitETHBridgeFinalized(
-        address _from,
-        address _to,
-        uint256 _amount,
-        bytes memory _extraData
-    )
-        internal
-        virtual
-    {
-        emit ETHBridgeFinalized(_from, _to, _amount, _extraData);
+        return
+            ((_mintableToken == l1USDC && _otherToken == l2USDC) || (_mintableToken == l2USDC && _otherToken == l1USDC));
```

Comment:
Can both case actually happen? I would have thought only `_mintableToken == l2USDC && _otherToken == l1USDC` can be true, but let's check in practice